.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and run tests
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
I don't know this workflow and possibilities of it, but `ubuntu` latest is quite big image that we do not need. Can you check if there is only node environment like on Docker? If it is, it is definitely smaller.

Reply:
It's the only Linux that [GitHub offers](https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources).

Reply:
Could we please freeze the version to `ubuntu-18.04`?

Otherwise they could add newer image and break everything at any time.